catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-schema-bindings/pom.xml

```diff
@@ -34,6 +34,11 @@
             <version>1.1.0</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jvnet.jaxb2_commons</groupId>
+            <artifactId>jaxb2-basics-runtime</artifactId>
+            <version>0.6.0</version>
```

Comment:
❓`0.6.0` is _really_ old. Can't we use `0.12.0` or `0.11.1`?

Reply:
No not for this one, it only works on this version for whatever reason. I tried all of the versions.